src/Goldilocks/Todo.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace Goldilocks;
+
+public class Todo
+{
+    public int Id { get; set; }
+
+    public string? Title { get; set; }
+
+    public DateOnly? DueBy { get; set; }
+
+    public bool IsComplete { get; set; }
+}
+
+static class TodoGenerator
+{
+    private static readonly (string[] Prefixes, string[] Suffixes)[] _parts = new[]
+        {
+            (new[] { "Walk the", "Feed the" }, new[] { "dog", "cat", "goat" }),
+            (new[] { "Do the", "Put away the" }, new[] { "groceries", "dishes", "laundry" }),
+            (new[] { "Clean the" }, new[] { "bathroom", "pool", "blinds", "car" })
+        };
+
+    internal static IEnumerable<Todo> GenerateTodos(int count = 5)
```

Comment:
Given this is a benchmark, perhaps we should remove the randomness here. Although it's only at application startup, the actual number of bytes in the todo titles will change from run to run which isn't ideal. Let's just make it return a hardcoded set of five todos.